tools/e2etesting/DeployPLCs.ps1

```diff
@@ -2,9 +2,25 @@ Param(
     [string]
     $ResourceGroupName,
     [int]
-    $NumberOfSimulations = 18,
```

Comment:
>18, [](start = 27, length = 3)

Please change back to 18, because this is the number of OPC UA server we have to verify

Reply:
Sorry, my mistake. Reverted.